ipaserver/install/krbinstance.py

```diff
@@ -372,6 +373,12 @@ def _wait_for_replica_kdc_entry(self):
             replication.wait_for_entry(remote_ldap, kdc_dn, timeout=60)
 
     def setup_pkinit(self):
+        try:
+            root_logger.debug("Adding '%s' CA to certmonger", PKINIT_CA_NAME)
+            certmonger.add_ca(PKINIT_CA_NAME, paths.IPA_PKINIT_SUBMIT)
+        except ValueError:
```

Comment:
This looks weird, a new exception should be used, like `CAAlreadyConfiguredError`